src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/OOPTagHelperResolver.cs

```diff
@@ -5,21 +5,29 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.ExternalAccess.Razor;
 using Microsoft.CodeAnalysis.Razor;
 using Microsoft.CodeAnalysis.Razor.ProjectSystem;
-using Microsoft.VisualStudio.Editor.Razor;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.VisualStudio.LanguageServices.Razor
 {
     internal class OOPTagHelperResolver : TagHelperResolver
     {
+        private static readonly JsonSerializer _serializer;
+
         private readonly DefaultTagHelperResolver _defaultResolver;
         private readonly ProjectSnapshotProjectEngineFactory _factory;
         private readonly ErrorReporter _errorReporter;
         private readonly Workspace _workspace;
 
+        static OOPTagHelperResolver()
```

Comment:
I made the serializer static because we  were going to be using it in both `ResolveTagHelpersOutOfProcessAsync` and `Serialize`, so I figured it was wasteful to re-create it every time.